client/helper.go

```diff
@@ -219,3 +224,76 @@ func AskForConfirmation(prompt string, tries int) bool {
 	}
 	return false
 }
+
+func ReadInteger(prompt string, tries int, min int, max int) (int, error) {
+	reader := bufio.NewReader(os.Stdin)
+	if tries <= 0 {
+		tries = 2
+	}
+
+	for ; tries > 0; tries-- {
+		fmt.Printf("%s [%d-%d]: ", prompt, min, max)
+
+		res, err := reader.ReadString('\n')
+		if err != nil {
+			log.Fatalf("error: %v", err)
```

Comment:
I guess this could be:

```suggestion
			return 0, fmt.Errorf("reading input failed: %w", err)
```

Reply:
I'm not entirely sure what error this handles, as I followed the pattern from the other function (the docs say "often io.EOF") but with your suggestion it will bail out on the error immediately, while this still logs the error and allows for retry.

Reply:
Let's keep it aligned with `AskForConfirmation` here. We can change it in another PR, as it will also affect other commands.